libs/shared/ui/multi-select.tsx

```
@@ -0,0 +1,616 @@
+                                  }}
+                                  className={cn(
+                                    'pl-8 cursor-pointer',
+                                    option.disable && 'cursor-default text-muted-foreground',
+                                  )}
+                                >
+
+                                  {option.label}
+                                </CommandItem>
+                              )
+                            })}
+                          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)